verilog/tools/kythe/kythe_facts.h

```diff
@@ -17,21 +17,60 @@
 
 #include <iosfwd>
 #include <string>
+#include <vector>
 
-#include "absl/strings/escaping.h"
 #include "absl/strings/string_view.h"
 
 namespace verilog {
 namespace kythe {
 
+// Unique identifier for Kythe facts.
+struct Signature {
```

Comment:
Should this be a class now, with public methods and private data member `names_`?